### ISSUE TITLE
6.3 WoT servient on Smartphone - Browser + Servient

### ISSUE THREAD

**Issue by mkovatsc** [w3c/wot-architecture#14]
The text for Figures 13 and 14 is quite confusing and it is hard to understand the difference (actually there is none described in the text). Also, the figure numbering needs fixing.

I assume 13 means the browser runs on a different device than the smartphone and 14 that the browser can also run on the smartphone itself. I don't see this as fundamentally different. Whether the smartphone offers its own sensors/actuators as Things or functions as gateway for other technologies such as Bluetooth is not depicted either (although maybe even more interesting).

I would just have one figure and explaining these options well.

**Comment by mmccool** [comment]
I rewrote these sections and I hope made this clearer.   The only difference as far as I can see is that in one case the browser runs on a different device, in the other it runs on the same device.   I did rip out some redundancy in the descriptions.

In practice, these are a bit different.   In particular, to get remote access to a Servient running on a phone, you would have to convince that phone to run a web server.   This is not that easy to do, and we probably should discuss access point modes, etc.   Then there's the issue of providing a global URL and DNS entry, pointing back at the phone, etc. etc.

Of course we could just remove one of these two use cases, too.
